deploy/mongodb/templates/clusterdefinition.yaml

```diff
@@ -33,10 +33,10 @@ spec:
           keys:
```

Comment:
check missing `componentDefs.workloadType` attribute, this is required attribute value, have you tested?

Reply:
workloadType: Consensus
      consensusSpec:
        leader:
          name: "primary"
          accessMode: ReadWrite